server/__init__.py

```diff
@@ -284,8 +285,10 @@ def create_app(nl_root=DEFAULT_NL_ROOT):
   if app.config['ENABLE_ADMIN']:
     register_routes_admin(app)
 
-  # Load place explorer summaries
-  app.config['PLACE_EXPLORER_SUMMARIES'] = libutil.get_place_summaries()
+  # Load place explorer summaries & allowlist of places to show summaries for
+  # Used when rendering place pages
+  app.config['PLACE_SUMMARY_ALLOW_LIST'] = place_summaries.get_place_allowlist()
+  app.config['PLACE_EXPLORER_SUMMARIES'] = place_summaries.get_place_summaries()
```

Comment:
This config is created on server start time. If we change the json content, needs to restart server. This does not require a release but should trigger a GKE pod restart by updating a dummy field in the yaml file.